### PR TITLE
Feature(EIP 831) - ethereum:address

### PR DESCRIPTION
**Pull request by justinkook**
- added support for EIP 831 ethereum:address
- fix sendMax totalAmount being negative
- Added translations for `{coinName} Address` and `Enter custom fee in {feeUnit}`
- Added feeUnit to choose-fee-level.ts - dynamic sat/byte or Gwei based on coin.

Specification details can be found here:
https://eips.ethereum.org/EIPS/eip-831

### CODE REVIEW

File: src/pages/send/choose-fee-level/choose-fee-level.html [hunk 1]
```diff
@@ -41,15 +41,15 @@
...
+            {{feePerSatByte}} {{ feeUnit }}
           </span>
         </ion-note>
       </ion-item>
     </ion-list>
 
     <ion-list class="fee-custom" *ngIf="customFeePerKB">
       <ion-item>
-        <ion-label color="primary" floating>{{'Enter custom fee in sat/byte' | translate}}</ion-label>
+        <ion-label color="primary" floating>{{'Enter custom fee in ' + feeUnit | translate}}</ion-label>
```
Comment (Gamboster) [inline src/pages/send/choose-fee-level/choose-fee-level.html]: Check this translate, I don't think it works as expected

**Comment by cmgustavo** [comment]
This feature is still a `draft`. Maybe we could wait for the next step (at least `accepted`) before to implement into the wallet

**Comment by Gamboster** [comment]
> This feature is still a `draft`. Maybe we could wait for the next step (at least `accepted`) before to implement into the wallet

Maybe @justinkook  can create two PRs: One with the fixes and another with the new feature, waiting for approval. What do you guys think?

**Comment by justinkook** [comment]
> > This feature is still a `draft`. Maybe we could wait for the next step (at least `accepted`) before to implement into the wallet
> 
> Maybe @justinkook can create two PRs: One with the fixes and another with the new feature, waiting for approval. What do you guys think?

All of the major ethereum wallets support EIP-831 that I've been testing so far:
- Coinbase wallet
- Exodus Wallet
- Blockchain Wallet
- Metamask

File: src/pages/send/choose-fee-level/choose-fee-level.html (hunk 1 above)
Comment (justinkook) [reply src/pages/send/choose-fee-level/choose-fee-level.html]: fixed

@Gamboster What do you think about doing it this way?:
```
{{'Enter custom fee in {feeUnit}' | translate: { feeUnit: feeUnit } }}
```

**Pull request merged**